pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -8,11 +8,12 @@ function omf.repo.pull
   set -l repo_dir $argv[1]
 
   function omf.repo.git -V repo_dir
-    command git --git-dir "$repo_dir"/.git --work-tree "$repo_dir" $argv
+    command git -C "$repo_dir" $argv
```

Comment:
Older version of OS X ships with Git < 1.8.5, which doesn't include `git -C` (see http://blogs.atlassian.com/2013/12/whats-new-git-1-8-5). Prefer keeping `--git-dir` with `--work-tree`.